include/faabric/mpi/mpi.h

```diff
@@ -183,8 +189,13 @@ extern "C"
     extern struct faabric_info_t faabric_info_null;
 #define MPI_INFO_NULL &faabric_info_null
 
+// Misc constants
+#define MPI_ANY_SOURCE -1
+#define MPI_UNDEFINED -1
```

Comment:
`MPI_UNDEFINED` is used as a rank for processes that have been left out of a _new_ communicator.

For instance, creatnig a 3x3 grid of processes and running with `-np 10` will make one be completely idle (this is expected behaviour).